src/goto-programs/builtin_functions.cpp

```diff
@@ -636,6 +636,88 @@ void goto_convertt::do_enum_is_in_range(
   copy(assignment, ASSIGN, dest);
 }
 
+void goto_convertt::do_havoc_slice(
+  const exprt &lhs,
+  const symbol_exprt &function,
+  const exprt::operandst &arguments,
+  goto_programt &dest)
+{
+  std::string identifier = CPROVER_PREFIX "havoc_slice";
+
+  // We disable checks on the generated instructions
+  // because we add our own rw_ok assertion that takes size into account
+  auto source_location = function.find_source_location();
+  source_location.add_pragma("disable:pointer-check");
+  source_location.add_pragma("disable:pointer-overflow-check");
+  source_location.add_pragma("disable:pointer-primitive-check");
+
+  // check # arguments
+  if(arguments.size() != 2)
+  {
+    error().source_location = source_location;
+    error() << "'" << CPROVER_PREFIX "havoc_slice"
+            << "' expected to have two arguments" << eom;
+    throw 0;
+  }
```

Comment:
We probably don't have any cases where we try to pass more than 2 parameters, thus, we don't hit this case.

Reply:
Since tests are expressed in C and uses of havoc_slice are checked by the front end these would be impossible to reach using a test expressed in C.
Covering them would require build and load a goto model directly or create them in C++ directly.